crates/test/src/lib.rs

```diff
@@ -57,11 +57,13 @@ pub fn trdelnik_test(args: TokenStream, input: TokenStream) -> TokenStream {
     let input_fn_span = input_fn.span();
     let input_fn_body = input_fn.block;
     let input_fn_name = input_fn.sig.ident;
+    let input_fn_inputs = input_fn.sig.inputs;
 
     quote::quote_spanned!(input_fn_span=>
+        #[trdelnik_client::rstest]
```

Comment:
I would also return back the code with `input_fn_attrs` so users can use standard Rust test attributes like
```rust
#[trdelnik_test]
#[ignore]
async fn test_happy_path(...
```
Related docs: https://doc.rust-lang.org/reference/attributes/testing.html